plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer_test.go

```diff
@@ -8,7 +8,8 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"go.opentelemetry.io/collector/pdata/pcommon"
-	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
+	deprecatedsemconv "go.opentelemetry.io/collector/semconv/v1.18.0"
+	conventions "go.opentelemetry.io/collector/semconv/v1.22.0"
```

Comment:
nit: keep the same pattern of calling it semconv as above

Reply:
Updated.